api/v1/clustercatalog_types.go

```diff
@@ -344,7 +349,7 @@ type ImageSource struct {
 	// When omitted, the image will not be polled for new content.
 	// +kubebuilder:validation:Minimum:=1
 	// +optional
-	PollIntervalMinutes *int `json:"pollIntervalMinutes,omitempty"`
+	PollIntervalMinutes *int `json:"pollIntervalMinutes,omitempty"` //nolint: kubeapilinter
```

Comment:
Nolint here because it wants this to be either `int32` or `int64` and not a pointer which would be a breaking change. Although I should probably remove the nolint